src/PriceOracle.sol

```diff
@@ -117,29 +121,35 @@ contract PriceOracle is AccessControl {
             uint32 _currentPrice = currentPrice;
             uint32 _minimumPrice = minimumPrice;
             uint32 _priceBase = priceBase;
+            uint64 _currentPriceUpscaled = currentPriceUpscaled;
```

Comment:
should be uint256

Reply:
24577265
24578108
24578529
24578529
some example values for it above

it is 8 digit number that is in
_currentPriceUpscaled

uint64 has 20 digits

p.s.
also if you worry about calculation maximums it goes around this number
1.2886892e+13  before division.  So there is still 6 digits left to spare and its a safe type to operate on the value.